vscode/src/jsonrpc/agent-protocol.ts

```diff
@@ -571,6 +570,8 @@ export interface ProtocolTextDocument {
         | {
               selectedText?: string | undefined | null
               sourceOfTruthDocument?: ProtocolTextDocument | undefined | null
+              workspaceDocumentURIs?: string[] | undefined | null
+              activeWorkspaceDocumentURI?: string | undefined | null
```

Comment:
It might be clearer to add a comment explaining the contract, even if "active document" is a variable name in the codebase
```suggestion
              // The document that the user is currently focusing on, which is defined by the latest `textDocument/{didOpen,didChange,didFocus}` event.
              activeWorkspaceDocumentURI?: string | undefined | null
```